shopelectro/tests/tests_commands.py

```diff
@@ -38,6 +39,27 @@ def get_tag_as_dict(group: str, tag: str):
     }
 
 
+@tag('fast')
+class UpdatePack(TestCase):
+
+    fixtures = ['dump.json']
+
+    def test_update_prices(self):
+        # @todo #859:30m Create fixture products with in_pack = 2
+
+        mul = 2
+        tags = Tag.objects.all()
+        tags.products().update(in_pack=mul)
+        products = list(tags.products())
+
+        update_pack.update_prices(tags)
+
+        for new, old in zip(tags.products(), products):
+            self.assertEqual(new.id, old.id)
+            for price in update_pack.PRICES:
+                self.assertEqual(getattr(new, price) / mul, getattr(old, price))
```

Comment:
```suggestion
                self.assertEqual(getattr(new, price), getattr(old, price) * mul)
```

it would be safer for comparing then float division